### PR TITLE
[WIP] Fixes: https://github.com/gitcoinco/web/issues/1855

### PR DESCRIPTION
**Pull request by saketbairoliya2**
##### Description
 This PR will handle creation of Bounty directly when a request is submitted. 

##### Checklist

- [x] linter status: 100% pass
- [ ] changes don't break existing behavior
- [ ] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Refers/Fixes

Refs: #1855

### CODE REVIEW

**Comment by mbeacom** [comment]
:star: PR built and available in a preview environment **gitcoinco-web-pr-2371** [here](http://web.jx-gitcoinco-web-pr-2371.ci.gitcoin.co)

File: app/bounty_requests/models.py [hunk 1]
```diff
@@ -64,3 +69,18 @@ class BountyRequest(SuperModel):
     def __str__(self):
         """Return the string representation of BountyRequest."""
         return f"{self.requested_by.username} / {self.created_on}"
+
+    def to_bounty(self, network=None):
```
Comment (mbeacom) [inline app/bounty_requests/models.py]: Should we associate the bountyrequest with the bounty?

**Comment by mbeacom** [comment]
@saketbairoliya2  Is this still a WIP?

**Comment by saketbairoliya2** [comment]
Yes, I'll try to finish it this week. Apologies for delay.

kumar saket

**Comment by octavioamu** [comment]
In general is looking good but need a rebase with the current explorer changes, lot of things have changed in the explorer

**Comment by kuhnchris** [comment]
@saketbairoliya2 @octavioamu i rebased it over at #3734

**Comment by thelostone-mc** [comment]
Closing this out based on above comment ^

**Pull request closed without merging**